template/config/storybook/stories.scss

```diff
@@ -0,0 +1,28 @@
+// @import "~highlight.js/styles/github.css";
```

Comment:
@taslangraham should these lines be commented out?

Reply:
I removed them